hail/python/hail/experimental/annotation_db.json

```diff
@@ -3,119 +3,119 @@
   "url": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4341060/",
```

Comment:
add hail-datasets-eu to hailctl/start.py

Reply:
Done